domain-models-runtime/src/main/java/org/folio/rest/RestVerticle.java

```diff
@@ -251,12 +251,13 @@ public void start(Promise<Void> startPromise) throws Exception {
     final BodyHandler handler = BodyHandler.create();
 
     // IMPORTANT!!!
-    // the body of the request will be read into memory for ALL PUT requests
+    // the body of the request will be read into memory for ALL PUT, ALL PATCH requests
```

Comment:
Can you update the documentation accordingly:
https://github.com/folio-org/raml-module-builder/blob/master/README.md

Reply:
Thanks!